packages/mobile-expo/Components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from "react";
+import { StyleSheet, Text, View } from "react-native";
+import { Dropdown } from "react-native-element-dropdown";
+
+type DropdownItem = {
+  label: string;
+  value: string;
+};
+
+type DropdownComponentProps = {
+  data: DropdownItem[];
+  handleChange: (item: string) => void;
+  label: string;
+  value: string;
+  isLoading: boolean;
+};
+
+const DropdownComponent = ({
+  data,
+  handleChange,
+  label,
+  value,
+  isLoading,
+}: DropdownComponentProps) => {
+  const [isFocus, setIsFocus] = useState(false);
+
+  const renderLabel = () => {
+    return (
```

Comment:
what is the purpose of doing this ??

Reply:
this code is present in the docs of react-native-element-dropdown which i'm using for dropdowns , but yeah i can put the label directly without even making a seperate arrow function